edb/schema/pointers.py

```diff
@@ -498,30 +552,44 @@ def get_is_derived(self, schema: s_schema.Schema) -> bool:
     def get_is_local(self, schema: s_schema.Schema) -> bool:
         return True
 
-    def get_union_of(self, schema):
+    def get_union_of(
+        self,
+        schema: s_schema.Schema,
+    ) -> Optional[so.ObjectSet[s_types.Type]]:
```

Comment:
```suggestion
    ) -> Optional[so.ObjectSet[PointerLike]]:
```

Reply:
Unfortunately this doesn't work because the Type argument of `ObjectSet` must be a subtype of "edb.schema.objects.Object". The `PseudoPointer` is not even an `objects.Object`.

Reply:
Among the first comments, this is the only one for which I didn't make a change in the current branch.

Reply:
It always returns `None`, so try annotating it as such.